src/main/java/com/garby/company/mapper/EmployeeRowMapper.java

```diff
@@ -0,0 +1,23 @@
+package com.garby.company.mapper;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import org.springframework.jdbc.core.RowMapper;
+
+import com.garby.company.entity.Employee;
+
+public class EmployeeRowMapper implements RowMapper<Employee> {
```

Comment:
The insertEmployee() method inserts an employee using template.update(sql,param,holder) where "param" is the SqlParameterSource, which maps the values dynamically in the query marked with a colon.